Resources/Auth/UserResource.cs

```diff
@@ -7,5 +8,6 @@ public class UserResource
         public int Id { get; set; }
         public string Email { get; set; }
         public IEnumerable<string> Roles { get; set; }
+        public AzureInformation AzureInformation { get; set; }
```

Comment:
Hvis vi bruk den her resourcen i `getAllUsers()` så får vi vel ut PAT for absolutt alle users, det e vel ikke så nødvendig å gjør.

Reply:
AzureInformation feltet dukka opp i getAllUsers ja, men den vil aldri ha noke data